packages/driver/src/cy/logGroup.ts

```diff
@@ -0,0 +1,57 @@
+import { $Command } from '../cypress/command'
+import $errUtils from '../cypress/error_utils'
+import type { LogConfig } from '../cypress/log'
+
+export interface LogGroupConfig {
```

Comment:
doesn't this extend `LogConfig` since we pass it to `Cypress.log`?

Reply:
@flotwig I purposely limited these options to keep this API clean and minimal. As additional use-cases are determined, we can expand this API.